ccd-definition/AuthorisationCaseEvent/DEPRECATED.json

```diff
@@ -335,5 +335,19 @@
     "CaseEventID": "uploadC2Gatekeeping",
     "UserRole": "[SOLICITOR]",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "amendChildren",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
```

Comment:
should be R for deprecated event. It should be not possible to create deprecated event